src/core/qgismobileapp.cpp

```diff
@@ -163,6 +163,13 @@ QgisMobileapp::QgisMobileapp( QgsApplication *app, QObject *parent )
   create();
 #endif
 
+  QSettings settings;
+  if ( PlatformUtilities::instance()->capabilities() & PlatformUtilities::AdjustBrightness )
+  {
+    mScreenDimmer.reset( new ScreenDimmer( app ) );
```

Comment:
```suggestion
    mScreenDimmer = std::make_unique<ScreenDimmer>( app );
```

Not that it would make any difference at all in the app itself, but let's get rid of `new` 
https://stackoverflow.com/questions/22571202/differences-between-stdmake-unique-and-stdunique-ptr-with-new